### PR TITLE
Adding IPv6 outputs

### PR DESCRIPTION
**Pull request by sc250024**
## Description
Adding the `ipv6_addresses` output available to `aws_instance` resources.

## Motivation and Context
Created because I want to create AAAA records for EC2 instances created by this module.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -15,6 +15,7 @@ locals {
   this_tags                         = coalescelist(aws_instance.this.*.tags, [""])
   this_volume_tags                  = coalescelist(aws_instance.this.*.volume_tags, [""])
   this_password_data                = coalescelist(aws_instance.this.*.password_data, [""])
+  this_ipv6_addresses               = compact(coalescelist(aws_instance.this.*.ipv6_addresses, [""]))
```
Comment (antonbabenko) [inline outputs.tf]: Are you sure that we need `compact` outside?

Comment (sc250024) [reply outputs.tf]: Not 100%, but I looked at the `compact` docs (https://www.terraform.io/docs/configuration/functions/compact.html), and thought that the function to remove unnecessary / empty items in the list couldn't hurt. The same was done for `this_public_ip` and `this_private_ip `, so I was keeping it consistent. If you know it's not needed, I can just remove it.

Comment (antonbabenko) [reply outputs.tf]: Probably it is not needed, but please verify it and remove it. The rest looks good.

**Comment by antonbabenko** [comment]
Hi Scott,

I am going to close this PR because this feature has been already implemented and available in [v2.11.0](https://github.com/terraform-aws-modules/terraform-aws-ec2-instance/releases/v2.11.0).

**Pull request closed without merging**